packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -76,9 +76,10 @@ const ExportResultAsCSVModal: FC<ExportResultAsCSVModalProps> = ({
     onClose,
     onConfirm,
 }) => {
-    const { data: resultData, isLoading } = useSavedChartResults(
+    const { data: resultData, isLoading } = useDashboardTileResults(
```

Comment:
if you combine these endpoints in the API, we could use one hook that optionally takes the filter field

Reply:
I don't feel strongly about combining them if you think we might want to apply different permissions to them in future!

Reply:
>  if you think we might want to apply different permissions to them in future!

That was my initial thought... but I don't think this will happen anytime soon, so let's keep it simple